bal_tools/graphql/core/liquid_pools_protocol_yield_fee.gql

```diff
@@ -18,7 +18,6 @@ query LiquidPoolsProtocolYieldFee {
                 { poolTypeVersion: 1 }
               ]
             }
-            { poolType_contains_nocase: "Gyro" }
```

Comment:
Is this saying that any Gyro pool charges a yield fee?

Reply:
this was the assumption in the old query, yes.

by removing the gyro just falls back into the first or case, meaning it needs to have a yield fee > 0 just like the rest